src/lib/lib_weechat.py

```diff
@@ -93,3 +97,12 @@ def apply_user_config(self):
         for line in self.charm_config['user-config'].strip().split('\n'):
             self.weechat_command(line)
         self.weechat_command('/save')
+
+    def encfs_mounted(self):
+        cmd = 'mount | grep {}'.format(self.weechat_folder)
+        try:
+            subprocess.check_output(cmd, shell=True)
+        except CalledProcessError:
```

Comment:
Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-weechat-0/.venv/lib/python3.6/site-packages/charms/reactive/__init__.py", line 74, in main
    bus.dispatch(restricted=restricted_mode)
  File "/var/lib/juju/agents/unit-weechat-0/.venv/lib/python3.6/site-packages/charms/reactive/bus.py", line 390, in dispatch
    _invoke(other_handlers)
  File "/var/lib/juju/agents/unit-weechat-0/.venv/lib/python3.6/site-packages/charms/reactive/bus.py", line 359, in _invoke
    handler.invoke()
  File "/var/lib/juju/agents/unit-weechat-0/.venv/lib/python3.6/site-packages/charms/reactive/bus.py", line 181, in invoke
    self._action(*args)
  File "/var/lib/juju/agents/unit-weechat-0/charm/reactive/weechat.py", line 41, in install_weechat
    if helper.charm_config['encfs-enabled'] and not helper.encfs_mounted():
  File "lib/lib_weechat.py", line 105, in encfs_mounted
    except CalledProcessError:
NameError: name 'CalledProcessError' is not defined

This isn't imported it seems, if you run the functional test on a local juju you'll reproduce.